priv/ie_gen_v1.erl

```diff
@@ -643,6 +643,9 @@ s2a(Name) ->
 	      end,
 	      string:to_lower(Name)).
 
+s2f(Flags) when is_list(Flags) ->
```

Comment:
If this pull request is to change from list to map, why keep this list?
A comment would be nice. (ie, not list of flags, but flag names?)

Reply:
The list of flags is used only in the code generator.